build/apis/contract-chain-c-chain-api.md

```diff
@@ -41,7 +41,459 @@ where `blockchainID` is the ID of the blockchain running the EVM.
 To interact with the `avax` specific RPC calls
 
 ```text
-/ext/bc/C/ava
+/ext/bc/C/avax
+```
+
+## AVAX RPC endpoints
+
+### avax.export
+
+Export an asset from the C-Chain to the X-Chain. After calling this method, you must call `import` on the X-Chain to complete the transfer.
+
+#### Signature
+
+```go
+avax.export({
+    to: string,
+    amount: int,
+    assetID: string,
+    username: string,
+    password:string,
+}) -> {txID: string}
+```
+
+* `to` is the X-Chain address the AVAX is sent to.
+* `amount` is the amount of nAVAX to send.
```

Comment:
Should be: 

"`amount` is the amount of the asset to send."

Since the asset might not be AVAX